src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -421,8 +421,9 @@ private OpenApiOperation GenerateOpenApiOperationFromMetadata(ApiDescription api
                 foreach (var content in requestContentTypes)
                 {
                     var requestParameters = apiDescription.ParameterDescriptions.Where(desc => desc.IsFromBody() || desc.IsFromForm());
-                    if (requestParameters is not null)
```

Comment:
This method is inefficient the way it's using LINQ.

First there's a check for "is there at least one", then there's a check for "is there exactly one" that is discarded, then they're counted, then there's either a `ToList()` if there isn't exactly one or `First()` is called three times to get the same value.

This can be refactored to be smarter.

Reply:
Done, I am not sure if I can remove the discard (To have the same behaviour as before)